Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py

```diff
@@ -77,12 +82,22 @@
     AutoInstall.register(Package('packaging', Version(20, 4)))
 
 AutoInstall.register(Package('pyparsing', Version(2, 4, 7)))
-AutoInstall.register(Package('requests', Version(2, 24)))
+
+if sys.version_info > (3, 7):
```

Comment:
Can we maybe change this to `3.9`? We will eventually need to sort out the `urllib3` import problem, but we can kick that can down the road until we're forced to address it.

Reply:
I'm going to change these all to 3.12 for the benefit of unbreaking git-webkit so I can continue normal WebKit development.

It's a real shame we don't understand why this is not working with python 3.9.